tests/minikube-config.yaml

```diff
@@ -40,4 +41,11 @@ jupyterhub:
       enabled: true
 
   debug:
-    enabled: true
\ No newline at end of file
+    enabled: true
+
+gateway:
+  proxyToken: "9e942640fd8ee885358e9d36991a22cccec58c8f855aebf7a95644f46d0eb9c0"
```

Comment:
@jcrist - CI is failing to find this value. Do you see an obvious typo that is messing me up here?

Reply:
When you include subcharts the chart values is joined at a key of the same name. Therefore the [gateway values file](https://github.com/dask/dask-gateway/blob/master/resources/helm/dask-gateway/values.yaml) will be dropped in here. 

So if you copy everything under `gateway:` and drop the indent it should match the file from the gateway chart.

It looks like the gateway chart also has a top level key called `gateway`. Which then contains the `proxyToken` key. This means you need that format in here too. Therefore I would expect the section to look like this:

```
gateway:
  gateway:
    proxyToken: "9e942640fd8ee885358e9d36991a22cccec58c8f855aebf7a95644f46d0eb9c0"
    auth:
      type: jupyterhub
      jupyterhub:
        apiToken: "6b9f64d7f421a3ee93fb64ba28c9284387e88893d2e1712c5ef1cdf56e565769"
```

Reply:
I think this whole section needs to be indented one level and put under `dask-gateway:`, since that's what you named the subchart?

Reply:
Ah, Jacob beat me to it :)

Reply:
Yeah but I messed up my example! Updated to be `dask-gateway`, thanks @jcrist !

Reply:
Thanks both of you. I've updated the chart and test config.